scripts/node_builder_glibc_env.sh

```diff
@@ -4,20 +4,24 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 set -eu
-# TODO support aarch64
-if [ "$(uname -m)" = "aarch64" ]; then
-    exit 0
-fi
 
-yum update -y && yum install -y curl python3 make git ca-certificates npm
+yum update -y && yum install -y curl make git ca-certificates npm
 # yum has node v10 by default, so we need to add newer version to run node-gyp
 curl -fsSL https://rpm.nodesource.com/setup_16.x | bash -
+yum remove -y nodejs npm
 yum install -y nodejs
 # node-gyp needs python 3.7+, so we need to install it
 yum -y install openssl-devel bzip2-devel
-curl -L https://www.python.org/ftp/python/3.7.9/Python-3.7.9.tgz -o Python-3.7.9.tgz
-tar xzf Python-3.7.9.tgz
-cd Python-3.7.9
-./configure --enable-optimizations
-make altinstall
-cd ..
\ No newline at end of file
+
+if [ "$(uname -m)" != "aarch64" ]; then
```

Comment:
In the x86_64 exe build we're also building Py 3.7, so we can use the same Python instead of installing? Not very important anyway.. maybe for a next PR or for when we finally get rid of the container build.

Reply:
centos/devtoolset-7-toolchain-centos7:latest image doesn't have python3 installed by default. We could use centos8 image and use static libstdc++ as well for x86_64, but I am not sure if it is the best way since centos 8 EOL.

Reply:
Wait.... also for x86_64 we don't need it anymore. We are building Python 3.7 for x86_64, but we have 3.8 in the container build and 3.10 in the exe build (since https://github.com/Granulate/gprofiler/pull/597), so why do we need to build 3.7?

Reply:
Added a comment to file node_builder_glibc_env.sh with a summary of when we use what image and python

Reply:
Good. Now that I have that in front of me - do we need to build 3.7.9 in ANY case?
* x86_64 container - is centos 7, can install python 3.6.8 like aarch64
* x86_64 exe - is centos 7, has the python 3.10 we built
* aarch64 container - is centos 8, use the python 3.6.8
* aarch64 exe - is centos 8, use the 3.10 we built

Reply:
Yep, I deleted code that built 3.7.9. One note -> in x86_64 exe we use now centos7 with python 3.6.8 (python 10 we are building on centos8 so we would need to use static libstdc++)